WORKFLOW.md

```diff
@@ -0,0 +1,69 @@
+# Grafana workflow
+
+This document is based on [GOVERNANCE.md](GOVERNANCE.md). We assume good faith and intend to keep all processes as lightweight as possible but as specific as required. In case of disagreements about anything in this document, GOVERNANCE.md applies.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC2119](http://tools.ietf.org/html/rfc2119).
+
+Git and [GitHub terminology](https://help.github.com/en/github/getting-started-with-github/github-glossary) are used throughout this document.
+
+Team members and their access to repositories is maintained through [GitHub teams](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-teams). Team maintainers add and remove team members as per GOVERNANCE.md
+
+
+# Code changes
+
+## Proposing changes
+
+Examples of proposed changes are overarching architecture, subsets, and specific code/graphical elements. Proposed changes SHOULD cover the big picture and intention, but individual parts SHOULD be split into the smallest possible changes. Changes SHOULD be based on and target the master branch. Depending on size of the proposed change, they SHOULD be discussed, in increasing order of change size and complexity:
+Directly in a Pull Request - this MAY be done, but SHOULD not be the common case
+Issue
+Developer mailing list
+Design document, shared via Google Docs, accessible to at least all team members
+
+Significant changes MUST be discussed and agreed upon with the relevant subsystem maintainers.
+
+## Merging Pull Requests
+Depending on the size and complexity of a Pull Request, different requirements MUST be applied. Any team member contributing substantially to a Pull Request MUST NOT count against review requirements.
+Commits MUST be merged into master using Pull Requests. They MUST NOT be merged into master directly.
+Every merge MUST be approved by at least one team member
+Non-trivial changes MUST be approved by at least
+two team members, or
+one subsystem maintainer
+Significant changes MUST be approved by at least
+two team members, AND
+the relevant subsystem maintainer
+
+Pull Requests MUST be [reviewed](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests) and [approved](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/approving-a-pull-request-with-required-reviews) via GitHub’s review system. Reviewers MAY write comments if approving and MUST write comments if rejecting a Pull Request or if requesting changes.
+Once a Pull Request is approved as per above, any team member MAY merge the Pull Request.
+
+
+# Release workflow
+
+## Branch structure
+Grafana uses trunk-based development.
+
+In particular, we found that the following principles match how we work
```

Comment:
Formatting is unclear. It looks like the following lines should be an unordered list. If so, please add the necessary formatting and end line 44 with a colon.